integration-tests/src/test/resources/features/action-slack.feature

```diff
@@ -40,3 +40,70 @@ Feature: Slack Action tests
     """
 
     Then Slack channel "${slack.channel.mc.name}" contains message with text "${cloud-event.my-id.id}" within 1 minute
+
+
+  Scenario: Slack Action Processor is created and slack webhook url getting updated and slack message should be received in the new slack channel
+    When add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+      "name": "slackProcessor",
+      "action": {
+        "type": "slack_sink_0.1",
+        "parameters": {
+            "slack_channel": "${slack.channel.general.name}",
+            "slack_webhook_url": "${slack.channel.general.webhook.url}"
+          }
+      }
+    }
+    """
+    And the Processor "slackProcessor" of the Bridge "mybridge" is existing with status "ready" within 5 minutes
+    And the Processor "slackProcessor" of the Bridge "mybridge" has action of type "slack_sink_0.1"
+
+    And send a cloud event to the Ingress of the Bridge "mybridge":
+    """
+    {
+      "specversion": "1.0",
+      "type": "Microsoft.Storage.BlobCreated",
+      "source": "StorageService",
+      "id": "test-id",
+      "time": "2019-11-18T15:13:39.4589254Z",
+      "subject": "blobServices/default/containers/{storage-container}/blobs/{new-file}",
+      "dataschema": "#",
+      "data": {
+        "name": "Hello world!"
+        }
+    }
+    """
+
```

Comment:
@sthundat here you should check that the message arrived to Slack `general` channel